src/techs/techs.service.ts

```diff
@@ -163,6 +167,143 @@ export class TechsService {
         }
     }
 
+    async updateExistingTeamTech(
+        req,
```

Comment:
req is of `CustomRequest` type if you want to add it ("../global/types/CustomRequest")